modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/view_kb_article.jsp

```diff
@@ -174,7 +174,7 @@ if (portletTitleBasedNavigation) {
 					>
 						<liferay-frontend:fieldset
 							collapsible="<%= false %>"
-							cssClass="panel-unstyled"
+							cssClass="knowledge-base-fieldset-margins panel-unstyled"
```

Comment:
There is only one `fieldset` in this page and the styles put on this could make sense in other fieldset, so in this case we could get rid of the new class (`knowledge-base-fieldset-margins`), and in css reference to the fieldset tag directly.